cookbook.go

```diff
@@ -296,12 +296,13 @@ func (cg *ChefGuard) getCookbookChangeDetails(r *http.Request) []byte {
 	}
 
 	details := fmt.Sprintf(
-		"{\"name\":\"%s\",\"version\":\"%s\",\"frozen\":%t,\"forcedupload\":%t,\"source\":\"%s\"}",
+		"{\"name\":\"%s\",\"version\":\"%s\",\"frozen\":%t,\"forcedupload\":%t,\"source\":\"%s\", \"updated_at\": \"%s\"}",
```

Comment:
I suggest to just call the new field `uploaded`. As this message says nothing about the content of the cookbook. So it could be updated, but is could also be the exact same cookbook being uploaded a second time.